meta-mel/conf/distro/mel.conf

```diff
@@ -68,7 +68,7 @@ META_MENTOR_DIR = "${LAYERDIR_mel}/.."
 PATH_prepend = "${META_MENTOR_DIR}/scripts:"
 
 # Default to the external toolchain if available
-TCMODE ?= "${@bb.utils.contains('BBFILE_COLLECTIONS', 'sourcery', 'external-sourcery-rebuild-libc', bb.utils.contains('BBFILE_COLLECTIONS', 'external-toolchain', 'external', 'default', d), d)}"
+TCMODE = "${@bb.utils.contains('BBFILE_COLLECTIONS', 'sourcery', 'external-sourcery-rebuild-libc', bb.utils.contains('BBFILE_COLLECTIONS', 'external-toolchain', 'external', 'default', d), d)}"
```

Comment:
How would this behave with the local.conf assignment?

Reply:
This is required as upstream now does a hard set of TCMODE *before* the distro .conf is parsed, and I have a second commit that adjusts local.conf.sample to define TCMODE_mel in the commented example, not just TCMODE, to ensure it takes precedence.